packages/eslint-plugin/CHANGELOG.md

```diff
@@ -5,6 +5,10 @@
 - New Rule: [`@wordpress/i18n-text-domain`](https://github.com/WordPress/gutenberg/blob/master/packages/eslint-plugin/docs/rules/i18n-text-domain.md)
 - New Rule: [`@wordpress/i18n-translator-comments`](https://github.com/WordPress/gutenberg/blob/master/packages/eslint-plugin/docs/rules/i18n-translator-comments.md)
 
+### Enhancements
```

Comment:
If it's the case that upgrading to a version that includes these changes would cause errors where there were none previously (which I believe it is), then I think we must classify this as a breaking change. I recognize that for this package in particular, that's going to qualify for a _lot_ of its changes, but I also don't think we're running out of major version numbers any time soon, so I don't necessarily see it as a problem.

Reply:
Works for me